mappings/net/minecraft/resource/ServerResourceManager.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5350 net/minecraft/resource/ServerResourceManager
+	FIELD field_25335 resourceManager Lnet/minecraft/class_3296;
+	FIELD field_25336 commandManager Lnet/minecraft/class_2170;
+	FIELD field_25337 recipeManager Lnet/minecraft/class_1863;
+	FIELD field_25338 registryTagManager Lnet/minecraft/class_3505;
+	FIELD field_25339 lootConditionManager Lnet/minecraft/class_4567;
+	FIELD field_25340 lootManager Lnet/minecraft/class_60;
+	FIELD field_25341 serverAdvancementLoader Lnet/minecraft/class_2989;
+	FIELD field_25342 functionManager Lnet/minecraft/class_5349;
+	METHOD <init> (ZI)V
+		ARG 1 isDedicatedServer
```

Comment:
```suggestion
		ARG 1 dedicated
```

Reply:
I will also change this to match in CommandManager